data/NishantCoder108.json

```diff
@@ -0,0 +1,32 @@
+{
+	"name": "Nishant Kumar",
+	"type": "personal",
+	"bio": "JavaScript | Open Source | Community Building ",
+	"links": [{
+			"name": "Follow me on GitHub",
+			"url": "https://github.com/NishantCoder108",
+			"icon": "FaGithub"
+		},
+		{
+			"name": "Follow me on Twitter",
+			"url": "https://twitter.com/NishantCoder",
+			"icon": "FaTwitter"
+		},
+     {
+			"name": "Follow me on LinkedIn",
+			"url": "https://www.linkedin.com/in/nishantcoder/",
+			"icon": "FaLinkedin"
+		},
+		{
+			"name": "Follow me on YouTube",
+			"url": "https://www.youtube.com/channel/UCrx_pTOxfgCLGAS7aYzG9iQ",
+			"icon": "FaYouTube"
```

Comment:
Please replace this with **' FaYoutube '** :+1:

Reply:
This is not a blocker so I will be merging this Pull Request.

Any changes or amendments to the Profile can be raised in future in a new Pull Request and you may wish to refer to the Editing Guide: https://linkfree.eddiehub.io/docs/how-to-guides/editing